lib/merchant.rb

```diff
@@ -0,0 +1,8 @@
+class Merchant
+  attr_reader :id,
+              :name
+  def initialize(merchant_data)
```

Comment:
Looks great!  However maybe we want the argument of merchant(merchant_data) to match the argument of item which is (data). Could do this later when refactoring.

Reply:
We can add spacing on line 3.5 later when we are working on final polishings of the project